app/presenters/features.rb

```diff
@@ -8,6 +8,10 @@ def self.wiki_ed?
     false
   end
 
+  def self.default_course_type
+    ENV['default_course_type'] || 'ClassroomProgramCourse'
+  end
+
```

Comment:
This is not used. (It could be used to replace the same method in DashboardPresenter; this is a better place for it.)